libfaasm/faasm/core.h

```diff
@@ -0,0 +1,222 @@
+#ifndef FAASMC_CORE_H
+#define FAASMC_CORE_H
+
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#define BYTES(arr) reinterpret_cast<uint8_t*>(arr)
+
+    /**
```

Comment:
How's that this declarations are indented?

Reply:
Hmm, good spot, I'll add clang-format etc. to this repo now that we have our own code in here